dig.go

```diff
@@ -357,12 +354,12 @@ func (c *Container) createParamObject(t reflect.Type) (reflect.Value, error) {
 			case "true", "yes":
```

Comment:
Side question: why are we supporting both of these values? If we want to support multiple values, why don't we just use `strconv.ParseBool` and accept all the usual truthy strings?